crates/pathfinder/src/state/sync/l2.rs

```diff
@@ -68,7 +64,15 @@ pub async fn sync(
         let block = loop {
             match download_block(next, &sequencer).await? {
                 DownloadBlock::Block(block) => break block,
-                DownloadBlock::AtHead => tokio::time::sleep(Duration::from_secs(5)).await,
+                DownloadBlock::AtHead => {
+                    use crate::ethereum::Chain;
+                    let sleep_duration = match chain {
+                        Chain::Mainnet => Duration::from_secs(60 * 15),
+                        Chain::Goerli => Duration::from_secs(30),
+                    };
```

Comment:
Can't we just extract this with some comments on why exactly these values were chosen? We seem to be duplicating this anyway.

I think I understand it now but I'm quite convinced I wouldn't  be able to do that without reading the conversation with Ariel...

Reply:
Fair point.